packaging/tests/tas-restapi/.travis.settings.xml

```diff
@@ -0,0 +1,10 @@
+<settings>

```

Comment:
Does this file have any meaning right now?

Reply:
No - we should delete this too.